Gemfile.lock

```diff
@@ -88,7 +88,7 @@ GEM
     high_voltage (3.1.0)
     i18n (1.2.0)
       concurrent-ruby (~> 1.0)
-    jquery-rails (4.2.1)
+    jquery-rails (4.3.3)
```

Comment:
@edwardloveall How do you feel about removing `jquery-rails`? We don't have any javascript in our app.

Reply:
I am for it!